docs/agile-development/advanced-topics/effective-organization/delivery-plan.md

```diff
@@ -22,7 +22,7 @@ One approach you can take to accomplish is with stickies and a spreadsheet.
 Step 1: Stack rank the features for everything in your backlog
 
 - Functional Features
-- [Non-functional Features] (docs/TECH-LEADS-CHECKLIST.md)
+- [Non-functional Features](docs/TECH-LEADS-CHECKLIST.md)
```

Comment:
This file no longer exists... the link check probably wasn't picking it up earlier due to the extra space...

Could you either a) remove the link altogether or if relevant, link to the non functional requirements section instead